src/lib/routes/setRequestData.js

```diff
@@ -0,0 +1,6 @@
+const setRequestData = (request) => {
+  const data = { ...request.body, ...request.params, ...request.query };
```

Comment:
Unit test this with all kinds of requests!
I feel like sometimes body, params and query can be undefined on different request methods and spreading on undefined will throw.
Check that!

Reply:
Hmm... I check: request body, params and query always exist like empty object, and even when they are undefined spreading doesn't throw.

Reply:
You're right, nice! I thought it throws on spreading undefined. Then it's fine to have it this way.

Reply:
Just add token here (and name token key `access_token`) from headers

Reply:
Just clarify, it's for future features? Or I should call this method directly in jwt-validator? Unfortunately jwt-validator works like a native middleware and placed up the chain of middlewares then dispatcher.
Now, I understand why it's not so good.

Reply:
Okay. I understand. Is It  for token validation to?